7_react/react1/week1/vite-project/src/App.jsx

```diff
@@ -0,0 +1,21 @@
+
+import React from 'react';
```

Comment:
this is more a fun fact, than an issue. In newer versions of React, you do not need to actually import `React` anymore